build-parsers.sh

```diff
@@ -28,5 +28,7 @@ fi
 
 #Generate the haxe and hxml parsers
 java -jar grammar-kit.jar gen ./grammar/haxe.bnf
-java -jar grammar-kit.jar gen ./src/com/intellij/plugins/haxe/hxml/hxml.bnf
+#java -jar grammar-kit.jar gen ./src/common/com/intellij/plugins/haxe/hxml/hxml.bnf
```

Comment:
I wonder, why?  My first guess is that there is an environment variable or a command line argument that IDEA passes along to Grammar-Kit.  Perhaps I can figure that out.

It's not that I prefer not to use the command line. Quite the opposite, in fact.  I would prefer to *never* check in the entire /gen tree, but to build it as part of the ant build process (which is also called when IDEA builds the project internally).  Also, just to be clear, I am *NOT* requesting that work as part of this change.